src/lib/countries/andorre.ts

```diff
@@ -0,0 +1,12 @@
+import { Country } from '../jsvat';
+
+export const andorre: Country = {
+  name: 'Andorre',
+  codes: ['AD', 'AND', '020'],
+  calcFn: (vat: string): boolean => {
+    return vat.length === 8;
```

Comment:
Hey, first of all thank you for your efforts ,

But I wonder - is that all the rules regarding VAT's number of Andorre?
I have nothing against including Andorre into the lib, but first I'd like to check if we can find any math rules to check Andorre's VATs

Reply:
I found info about it [here](https://www.oecd.org/tax/automatic-exchange/crs-implementation-and-assistance/tax-identification-numbers/Andorra-TIN.pdf).
I can change the rule in order to check if the first letter is on of this 'FELACDGOPU'.
The second letter(control letter) i've read in other github repo that is not documented for the oecd.org.

Reply:
Well, it's definitely better the nothing
I'd prefer to check the control letter as well, if it's possible, but I understand that it might be problematic